lib/schemas.js

```diff
@@ -21,6 +21,7 @@ const HANDLER_OPTIONS = joi.object({
   idField: [FIELD, joi.array().min(1).items(FIELD)],
   index: joi.string().min(1),
   indexField: [FIELD, joi.array().min(1).items(FIELD)],
+  indexPrefix: joi.string().allow(''),
```

Comment:
This requires me to use validated options instead of original ones. Thought about this already, but will do with the next changes.